CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.7.0+1
+ 
+ * Upgrade dependency on `_discoveryapis_commons`, changing `ApiRequestError`
```

Comment:
The lower bound still allows the old behaviour, that's a bit hard to read from this changelog